### PR TITLE
Supporting alternate qualities in act_int

### PR DESCRIPTION
**Pull request by Wires77**

### CODE REVIEW

File: Data/3_0/Skills/act_int.lua [hunk 1]
```diff
@@ -4682,6 +4756,11 @@ skills["IceNova"] = {
 	skillTypes = { [SkillType.Spell] = true, [SkillType.Hit] = true, [SkillType.Area] = true, [SkillType.SkillCanTrap] = true, [SkillType.SkillCanTotem] = true, [SkillType.SkillCanMine] = true, [SkillType.SpellCanRepeat] = true, [SkillType.Triggerable] = true, [SkillType.ColdSkill] = true, [SkillType.CanRapidFire] = true, [SkillType.AreaSpell] = true, [SkillType.NovaSpell] = true, },
 	statDescriptionScope = "skill_stat_descriptions",
 	castTime = 0.7,
+    statMap = {
```
Comment (LocalIdentity) [inline Data/3_0/Skills/act_int.lua:4759]: Should specify that it only applies to hits

File: Data/3_0/Skills/act_int.lua [hunk 2]
```diff
@@ -5261,6 +5345,12 @@ skills["KineticBolt"] = {
 		["active_skill_additive_spell_damage_modifiers_apply_to_attack_damage_at_%_value"] = {
 			flag("ImprovedSpellDamageAppliesToAttacks"),
 		},
+		["cast_speed_+%_applies_to_attack_speed_at_%_of_original_value"] = {
```
Comment (LocalIdentity) [inline Data/3_0/Skills/act_int.lua:5348]: This relies on this PR (https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/1364) being merged, so use the same format as the one in the PR

File: Data/3_0/Skills/act_int.lua [hunk 3]
```diff
@@ -6228,6 +6334,11 @@ skills["PowerSiphon"] = {
 	},
 	statDescriptionScope = "skill_stat_descriptions",
 	castTime = 1,
+    statMap = {
+        ["critical_ailment_dot_multiplier_+"] = {
```
Comment (LocalIdentity) [inline Data/3_0/Skills/act_int.lua:6338]: Should be in SkillStatMap.lua

File: Data/3_0/Skills/act_int.lua [hunk 4]
```diff
@@ -6393,6 +6504,9 @@ skills["Sanctify"] = {
 		["sanctify_wave_damage_+%_final"] = {
 			mod("Damage", "MORE", nil, 0, 0, { type = "SkillPart", skillPart = 2 }),
 		},
+		["consecrated_ground_enemy_damage_taken_+%"] = {
```
Comment (LocalIdentity) [inline Data/3_0/Skills/act_int.lua:6507]: Should be in SkillStatMap.lua

File: Data/3_0/Skills/act_int.lua [hunk 5]
```diff
@@ -6948,6 +7074,13 @@ skills["RaiseZombie"] = {
 		["zombie_slam_area_of_effect_+%"] = {
 			mod("MinionModifier", "LIST", { mod = mod("AreaOfEffect", "INC", nil, 0, 0, { type = "SkillId", skillId = "ZombieSlam" }) }),
 		},
+		["minion_life_regeneration_rate_per_minute_%"] = {
```
Comment (LocalIdentity) [inline Data/3_0/Skills/act_int.lua:7077]: Both should be in SkillStatMap.lua

**Review by LocalIdentity — CHANGES_REQUESTED** [review]
Also would appreciate if you used tabs over spaces for consistency with other parts of the codebase

File: Data/3_0/Skills/act_int.lua [hunk 6]
```diff
@@ -2354,6 +2383,12 @@ skills["Discipline"] = {
 		["base_maximum_energy_shield"] = {
 			mod("EnergyShield", "BASE", nil, 0, 0, { type = "GlobalEffect", effectType = "Aura" }),
 		},
+		["damage_+%_on_full_energy_shield"] = {
```
[the remark this answers is not in the document]
Comment (Wires77) [reply Data/3_0/Skills/act_int.lua:2386]: I just realized these don't have `{ type = "GlobalEffect", effectType = "Aura" }`  Do they still need to go in SkillStatMap?  What's the guideline for which place they should go, if it's not skill-specific, put it in SkillStatMap?

File: Data/3_0/Skills/act_int.lua [hunk 7]
```diff
@@ -3800,6 +3855,11 @@ skills["FreezingPulse"] = {
 		activeSkill.skillModList:NewMod("Damage", "MORE", -50, "Skill:FreezingPulse", { type = "DistanceRamp", ramp = {{0,0},{60*output.ProjectileSpeedMod,1}} })
 		activeSkill.skillModList:NewMod("EnemyFreezeChance", "BASE", 25, "Skill:FreezingPulse", { type = "DistanceRamp", ramp = {{0,1},{15*output.ProjectileSpeedMod,0}} })
 	end,
+	statMap = {
```
[the remark this answers is not in the document]
Comment (Wires77) [reply Data/3_0/Skills/act_int.lua:3858]: Hmm, I suppose that would be the same thing as 20% less damage, but yeah I'll take it out until the ramp calculation is added regardless

**Review by LocalIdentity — APPROVED** [review]

**Pull request merged**